Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM aligent/serverless:latest
+
+EXPOSE 9229
```

Comment:
Is there a reason we can't just add this to the `aligent/serverless` dockerfile?

Reply:
Well, I cannot think of any reason why we can't do that. Will make a PR for it.